scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -12,32 +12,35 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public static class PublishTargets
     {
-        private static CloudBlobContainer BlobContainer { get; set; }
+        private static AzurePublisher AzurePublisherTool { get; set; }
+
+        private static DebRepoPublisher DebRepoPublisherTool { get; set; }
 
         private static string Channel { get; set; }
 
-        private static string Version { get; set; }
+        private static string CliVersion { get; set; }
 
-        private static string NuGetVersion { get; set; }
+        private static string CliNuGetVersion { get; set; }
 
+        private static string SharedFrameworkNugetVersion { get; set; }
 
         [Target]
         public static BuildTargetResult InitPublish(BuildTargetContext c)
         {
-            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(Environment.GetEnvironmentVariable("CONNECTION_STRING").Trim('"'));
-            CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
-            BlobContainer = blobClient.GetContainerReference("dotnet");
+            AzurePublisherTool = new AzurePublisher();
+            DebRepoPublisherTool = new DebRepoPublisher(Dirs.Packages);
 
-            Version = c.BuildContext.Get<BuildVersion>("BuildVersion").SimpleVersion;
-            NuGetVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion;
+            CliVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").SimpleVersion;
+            CliNuGetVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion;
+            SharedFrameworkNugetVersion = c.BuildContext.Get<string>("SharedFrameworkNugetVersion");
             Channel = c.BuildContext.Get<string>("Channel");
+
             return c.Success();
         }
 
         [Target(nameof(PrepareTargets.Init),
         nameof(PublishTargets.InitPublish),
-        nameof(PublishTargets.PublishArtifacts),
-        nameof(PublishTargets.TriggerDockerHubBuilds))]
```

Comment:
Why was this removed?  I'm not finding it anywhere else?  Docker builds are no longer being triggered.